src/Questions.jsx

```diff
@@ -37,14 +39,23 @@ import './App.css'
            */
 const Questions = ({ questions, start, setStart }) => {
 
+    // You have quite a bit of state here
+    // It would be better if you could take your time to
+    // think what would be the best model to store each question into
+    // (hint: you are free to manipulate the incoming API data)
+
+
     /*  Answers state for storing all the shuffled answer*/
-    const [answers, setAnswers] = React.useState([]);
+    const [answers, setAnswers] = React.useState([]); // <--- this won't be needed once you got your questions right
     /*const [selectedAnswers, setSelectedAnswers] = React.useState({ qIndex: null, useranswer: null })*/
-    const [selectedAnswers, setSelectedAnswers] = React.useState(Array(questions.length).fill(null))
-    const [correctCount, setCorrectCount] = React.useState(0)
+    const [selectedAnswers, setSelectedAnswers] = React.useState(Array(questions.length).fill(null)) // <--- this won't be needed once you got your questions right
+    const [correctCount, setCorrectCount] = React.useState(0) // <--- this won't be needed once you got your questions right
```

Comment:
need assistance

Reply:
Let's work on the main state itself, then we can cover these values.